backend/src/packages/journal-entries/journal-entry.service.ts

```diff
@@ -46,8 +50,20 @@ class JournalEntryService implements Service {
     return Promise.resolve(null);
   }
 
-  public delete(): ReturnType<Service['delete']> {
-    return Promise.resolve(true);
+  public async delete(
+    id: number,
+    user: UserAuthResponseDto,
+  ): ReturnType<Service['delete']> {
+    // const journal = await this.find(); //TODO find(id)
+    const journal = { id: 1, userId: 32 };
+    if (journal.userId !== user.id) {
+      throw new AuthError({
+        status: HTTPCode.NOT_FOUND,
+        message: ExceptionMessage.INCORRECT_CREDENTIALS,
```

Comment:
It's not a suitable error

Reply:
@v0ldemar01 should it be bad BADREQUEST?

Reply:
I mean AuthError

Reply:
done

Reply:
It is not appropriate message: ExceptionMessage.INCORRECT_CREDENTIALS